charts/kof-storage/values.yaml

```diff
@@ -43,9 +43,6 @@ victoriametrics:
 grafana:
   enabled: true
   datasources:
-    - name: metrics
```

Comment:
We shouldn't do that. On remote clusters we don't create a promxy server group, so this is the only datasource available for a local grafana to get metrics.